app/jobs/property_impressions_drop_notification_job.rb

```diff
@@ -0,0 +1,38 @@
+# Alerts the team when a property has a dramatic drop in impressions
+# Scheduled via Heroku Scheduler
+# SEE: lib/tasks/schedule.rake
+class PropertyImpressionsDropNotificationJob < ApplicationJob
+  queue_as :low
+
+  def perform
+    ScoutApm::Transaction.ignore! if rand > (ENV["SCOUT_SAMPLE_RATE"] || 1).to_f
+    properties = []
+
+    Property.active.find_each do |property|
+      next unless property.daily_summary_reports_by_campaign(2.days.ago, 1.day.ago).sum(&:impressions_count) < 5
+      next unless property.daily_summary_reports_by_campaign(3.days.ago, 2.day.ago).sum(&:impressions_count) > 5
+      properties << property
+    end
```

Comment:
I might move the threshold to a variable. Also, `5` seems low for this check. I'd consider `10` or `25` given that most properties that meet our traffic requirements should be getting at least 300 requests per day.

Reply:
@jdorfman ^

Reply:
@hopsoft @andrewmcodes sounds good